tests_common/pytest_plugin.py

```diff
@@ -834,6 +834,7 @@ def __exit__(self, type, value, traceback):
                 return
 
             dag.clear(session=self.session)
+            dag.relative_fileloc = dag.fileloc
```

Comment:
This probably needs to happen in the AF3 block below?

Reply:
it does not seem to've caused a problem in any of the tests?

Reply:
hmm, weird! I guess in an old version you'd just be swapping the method for a string, which is "fine" I guess. I'd still move it.

Reply:
as you wish